content/maxwell1_fundamentals/dipole_sources_in_homogeneous_media/index.rst

```diff
@@ -3,37 +3,93 @@
+
+
+.. figure:: images/dipoles.png
+		:align: center
+		:figwidth: 100%
+		:name: DipolesFig
+		
+		(Left) Electrical current dipole ( :math:`\mathbf{p}`) oriented in the :math:`\hat x` direction and the primary current density ( :math:`\mathbf{J}`) it produces. (Right) Magnetic dipole ( :math:`\mathbf{m}`) oriented in the :math:`\hat y` direction and the primary magnetic field ( :math:`\mathbf{H}`) it produces.
+
+
+Both electrical current dipoles and magnetic dipoles can be represented as source terms in Maxwell's equations.
+For the electrical current dipole, the source term (:math:`\mathbf{J_e^s}`) represents an electrical current density and has units A/m :math:`\!^2`.
+For the magnetic dipole, the source results from a magnetization (:math:`\mathbf{M}`).
+However, the magetic source term (:math:`\mathbf{J_m^s} = - i\omega \mu \mathbf{M}`) is generally represented by a magnetic current density, where :math:`\mathbf{J_m^s}` has units V/m :math:`\!^2`.
+Details regarding the electric and magnetic current densities were provided along with the :ref:`Ampere-Maxwell equation<ampere_maxwell>`.
+
+In the presence of an electromagnetic source, Maxwell's equations in the frequency domain are given by:
+
+.. math::
+	\begin{split}
+	\nabla \times \mathbf{E} + i \omega & \mu \mathbf{H} = \pm \, \mathbf{J_m^s}  \\
+	\nabla \times \mathbf{H} - (\sigma + & i \omega \varepsilon ) \mathbf{E} = \pm \, \mathbf{J_e^s}
+	\end{split}
+    :name: eq_Maxwells_frequency
 
 
+where :math:`\pm` depends on a choice in sign convention.
+Equivalently, Maxwell's equations in the time domain are given by:
+
 .. math::
 	\begin{split}
-	\nabla \times \mathbf{E} + i \omega \mu \mathbf{H} &= \pm \, \mathbf{J_m}  \\
-	\nabla \times \mathbf{H} - \sigma \mathbf{E} &= \pm \, \mathbf{J_e}
+	\nabla \times \mathbf{e} \, + \, &\mu \dfrac{\partial  \mathbf{h} }{\partial t} = \pm \mathbf{j_m^s} \\
+	\nabla \times \mathbf{h} - \sigma \mathbf{e} &  - \varepsilon \dfrac{\partial \mathbf{e} }{\partial t} = \pm \mathbf{j_e^s}
 	\end{split}
+    :name: eq_Maxwells_time
 
 
+where :math:`\mathbf{j_e^s}` and :math:`\mathbf{j_m^s}` are time-dependent electric and magnetic source terms, respectively.
+Ultimately, the right-hand side of :ref:`Faraday's law <faraday_differential_frequency>` becomes non-zero in the presence of a magnetic source.
+And the right-hand side of the :ref:`Ampere-Maxwell law <ampere_maxwell_differential_frequency>` becomes non-zero in the presence of an electrical current source.
 
 
 **Contents**
 
+For each dipole source, we begin by presenting a physical model.
+This is used to replace frequency-dependent or time-dependent source terms in Maxwell's equations.
+Analytic and asymptotic expressions along with numerical modeling tools are then provided.
+Quick links to subsequent content are given below.
+
+
+**Electrical Current Dipole**
+
 .. toctree::
     :maxdepth: 2
 
+    electric_dipole_definition/index
```

Comment:
here, the naming is inconsistent between `electric dipole` and `electric current dipole`. For brevity, I would vote for sticking with `electric dipole` in the table of contents / titles. 
What do you think? 
![image](https://cloud.githubusercontent.com/assets/6361812/23597967/fde2ce5c-01eb-11e7-9677-85d18b839874.png)